plugins/com.google.cloud.tools.eclipse.appengine.whitelist/plugin.xml

```diff
@@ -1,4 +1,11 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <?eclipse version="3.4"?>
 <plugin>
+  <extension
+      id="appEngineWhitelist"
+      point="org.eclipse.core.resource.builders">
+    <builder hasNature="true">
```

Comment:
I think `hasNature="true"` may mean we can't add the builder to a project via WTP facets?

That said, I think we can have a WTP facet add/remove a nature from the project.